frontend/app/templates/search.html

```diff
@@ -7,6 +7,12 @@
     </div>
 
     <div id="schools-wrapper" data-ng-show="all_schools.length">
+        <!-- Legend -->
+        <div id="schools-legend">
+            <div class="schools-legend-el"><img src="./img/magnet.png">Magnet</div>
+            <div class="schools-legend-el"><img src="./img/calendar.png">Year-round</div>
+            <div class="schools-legend-el"><img src="./img/scroll.png">Charter</div>
```

Comment:
It might be nice to make the icons more uniform to match the look/feel of the surrounding interface. Maybe from the NounProject: [Magnets](https://thenounproject.com/search/?q=magnet), [Charters](https://thenounproject.com/search/?q=scroll), and [Year-Round](https://thenounproject.com/search/?q=calendar). They could still be colored. What do you think?